main.go

```diff
@@ -298,26 +304,29 @@ var RewriteCommand = cli.Command{
 
 		VLog("  - building rewrite mapping")
 		mapping := make(map[string]string)
-		if !c.Args().Present() {
-			err = buildRewriteMapping(pkg, pkgdir, mapping, c.Bool("undo"))
-			if err != nil {
-				return fmt.Errorf("build of rewrite mapping failed:\n%s", err)
-			}
-		} else {
+
+		err = buildRewriteMapping(pkg, pkgdir, mapping, c.Bool("undo"))
+		if err != nil {
+			return fmt.Errorf("build of rewrite mapping failed:\n%s", err)
+		}
+		if c.Args().Present() {
+			keepSet := map[string]struct{}{}
 			for _, arg := range c.Args() {
-				dep := pkg.FindDep(arg)
-				if dep == nil {
-					return fmt.Errorf("%s not found", arg)
+				keepSet[arg] = struct{}{}
+			}
+			undo := c.Bool("undo")
```

Comment:
Let's move this up and have `buildRewriteMapping` and this added logic refer the local `bool` variable.